python/dask_cudf/dask_cudf/backends.py

```diff
@@ -345,3 +346,14 @@ def group_split_cudf(df, c, k, ignore_index=False):
             ),
         )
     )
+
+
+@sizeof_dispatch.register_lazy("cudf")
+def register_cudf():
+    @sizeof_dispatch.register((cudf.DataFrame))
+    def sizeof_cudf_dataframe(df):
+        return int(df.memory_usage().sum())
+
+    @sizeof_dispatch.register((cudf.Series, cudf.BaseIndex))
+    def sizeof_cudf_series_index(obj):
+        return obj.memory_usage()
```

Comment:
No need to make the registration *lazy*:
```suggestion
@sizeof_dispatch.register(cudf.DataFrame)
def sizeof_cudf_dataframe(df):
    return int(df.memory_usage().sum())


@sizeof_dispatch.register((cudf.Series, cudf.BaseIndex))
def sizeof_cudf_series_index(obj):
    return obj.memory_usage()
```

Reply:
All non-standard Python objects are registered lazily in Dask: https://github.com/dask/dask/blob/main/dask/sizeof.py#L94-L217 .

Reply:
> All non-standard Python objects are registered lazily in Dask: https://github.com/dask/dask/blob/main/dask/sizeof.py#L94-L217 .

True, but that is to avoid having to import cudf, cupy, numba, immediately since they might not be available. `register_lazy("cudf")` is basically saying: if the user happens to import `cudf`, please register the following dispatch types. 

Notice, all the existing dispatching in this file, such as `group_split_dispatch` and `union_categoricals_dispatch`, also use `register()`.

Reply:
Yeah, good point. +1 for that.

Reply:
Thanks for that context @madsbk.